src/applications/disability-benefits/all-claims/pages/vaMedicalRecords.js

```diff
@@ -57,10 +59,20 @@ export const uiSchema = {
           required: 'Please select at least one condition',
         },
       },
-      treatmentDateRange: dateRangeUI(
-        'When did you first visit this facility?',
-        'When was your most recent visit?',
-        'Date of last treatment must be after date of first treatment',
+      treatmentDateRange: merge(
+        {},
+        dateRangeUI(
+          'When did you first visit this facility?',
+          'When was your most recent visit?',
+          'Date of last treatment must be after date of first treatment',
+        ),
+        {
+          from: {
+            'ui:validations': dateRangeUI().from['ui:validations'].concat([
```

Comment:
`.concat`'s fine, but you could alternatively `.push` this onto the array since there's only one.

Reply:
So actually `.push()` returns the new length of the array, so I don't think this would work, unless I set it up somewhere else, which I think is messier than the `concat` approach.

Reply:
Blech. Right. Forgot about that. Sounds like `.concat` is the way to go. :+1: